### PR TITLE
Allow phoenix version >= 0.16.0 and phoenix_html >= 2.1.0

### PR DESCRIPTION
**Pull request by seanski**
This pull request bumps the versions of phoenix and phoenix_html.

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -20,7 +20,7 @@ defmodule PhoenixSlim.Mixfile do
   end
 
   def deps do
-    [{:phoenix, "~> 0.16"},
+    [{:phoenix, "~> 1.0.0"},
```
Comment (doomspork) [inline mix.exs]: Should we make this something like `{:phoenix, "> 0.16.0"}`?  Are there any disadvantages?  The upside is fewer releases.

Comment (seanski) [inline mix.exs]: I _think_ it should be fine, as long as the API for Phoenix.Template.Engine doesn't change.

**Comment by doomspork** [comment]
Thanks @seanski!

**Comment by seanski** [comment]
How's this?

**Comment by seanski** [comment]
Ah, you're welcome!

**Pull request merged**